web/assets/js/dashboard.js

```diff
@@ -187,12 +187,15 @@ function loadTableData(tableTarget, sourceData, colTitle) {
     // load scores
     var tableBody = $('#' + tableId + '-body');
     $($(sourceData).get().reverse()).each(function(index, row) {
-        var newRow = $('<tr>');
-        newRow.append($('<td>').text(row['name']));
-        $(row['y']).each(function(i, count) {
-            newRow.append($('<td>').text(count));
-        });
-        tableBody.append(newRow);
+        // Ignore row if no data provided
+        if (row['y'].length) {
```

Comment:
I know we don't have an official style guide/linter, and this is probably mostly code you didn't write, but when referencing an object's property, use `.` instead of `[]`.

`row.y` and `row.name`, and `sourceData[0].x` earlier in this method

Reply:
Done.